plugins/legacy-plugin-chart-country-map/src/countries.ts

```diff
@@ -79,6 +81,9 @@ export const countryOptions = Object.keys(countries).map(x => {
   if (x === 'uk' || x === 'usa') {
     return [x, x.toUpperCase()];
   }
+  if (x === 'italy_regions') {
+    return [x, 'Italy (regions)'];
```

Comment:
I believe this should be translated:
```suggestion
    return [x, t('Italy (regions)')];
```

Reply:
If I'm not wrong, there aren't translations for country map names. I think adding translations should be another task.

Reply:
Fair enough - let's do a translation sweep later when that becomes a priority.